docs/topics/api/ratings.rst

```diff
@@ -78,12 +79,16 @@ This endpoint allows you to fetch a rating by its id.
 
     .. _rating-detail-object:
 
+    :query string include_flags_for: The user id to show flags for. If the request is made with an authenticated user matching this parameter value, a ``flags`` property will be added to the response as described below.
     :>json int id: The rating id.
     :>json object addon: A simplified :ref:`add-on <addon-detail-object>` object that contains only a few properties: ``id``, ``name``, ``icon_url`` and ``slug``.
     :>json string|null body: The text of the rating.
     :>json boolean is_deleted: Boolean indicating whether the rating has been deleted or not.
     :>json boolean is_latest: Boolean indicating whether the rating is the latest posted by the user on the same add-on.
     :>json int previous_count: The number of ratings posted by the user on the same add-on before this one.
+    :>json object flags[]: A list of flags the user requesting has previously applied to this add-on (that haven't been processed by moderators already).  In the current implementation the list will only ever contain one item, or be an empty list if there are no flags from the user.  Only present if ``include_flags_for`` parameter sent.
```

Comment:
Saying that the list will only ever contain one item seems unimportant to me.

Reply:
https://github.com/mozilla/addons-server/pull/9988/commits/f4bf3d514e65fb61a6dafafc552539de0bcfadc3